training/model_management/components/download_model/asset.yaml

```diff
@@ -1,2 +1,3 @@
 type: component
 spec: spec.yaml
+categories: ["Import model"]
```

Comment:
Need to use as per Louie's suggestion:
`categories: ["Models"]`

Reply:
Done